### PR TITLE
Refactor config

### PR DESCRIPTION
**Pull request by JonLazariniDD**
## Ticket title
One line description
[link to ticket number]()

## Checklist
- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by alleypa — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -40,3 +40,8 @@ UserInterfaceState.xcuserstate
 env
 *.env
 sentry.properties
+
+# config files
+config/**/*
+
+TESTS-PhantomJS_*
```
Comment (alleypa) [inline .gitignore:47]: remove teminals

File: package.json [hunk 2]
```diff
@@ -183,4 +187,4 @@
       "pre-push": "npm run build && npm run test-coverage"
     }
   }
-}
+}
```
Comment (alleypa) [inline package.json:190]: remove terminals

File: scripts/config/application.hybrid.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "BACKEND_URL_LOGS": "${BASE_URL}/${ENV}/logs",
+    "BACKEND_URL_VISIT": "${BASE_URL}/${ENV}/activities",
+    "BACKEND_URL_UNAUTH_LOGS": "${BASE_URL}/${ENV}/unauth-logs",
+    "URL_LATEST_VERSION": ""
+  },
+  "sentry": {
+    "SENTRY_ENV": "${environment}",
+    "SENTRY_DSN": "${SENTRY_DSN}",
+  }
+}`
```
Comment (alleypa) [inline scripts/config/application.hybrid.ts:41]: remove terminals everywhere

File: scripts/create-configs.ts [hunk 4]
```diff
@@ -1,14 +1,21 @@
+/**
+ * This file is created to be able to dynamically create config files per envs
+ * Files will be created with npm run config.
+ * Please don't forget to rename your env file to .env with the appropriate env vars
+ */
+require('dotenv').config();
```
Comment (alleypa) [inline scripts/create-configs.ts:6]: because we have commonJS, try to remove this. Use a normal import

**Comment by JonLazariniDD** [comment]
Will be part of 18854

**Comment by JonLazariniDD** [comment]
closing as it is incorporated in #396

**Pull request closed without merging**